docker/Dockerfile.dev

```diff
@@ -49,7 +49,7 @@ RUN apt-get update && \
 RUN pip install git+https://github.com/bacpop/PopPUNK@${POPPUNK_VERSION}#egg=PopPUNK 
 
 # Poetry setup
-RUN pip install poetry
+RUN pip install poetry==1.8.3
```

Comment:
Again, maybe just mention reason for this pin

Reply:
as above